build.gradle

```diff
@@ -474,6 +475,23 @@ project('spring-integration-cassandra') {
     }
 }
 
+project('spring-integration-cloudevents') {
+    description = 'Spring Integration CloudEvents Support'
+
+    dependencies {
+        api "io.cloudevents:cloudevents-core:$cloudEventsVersion"
+        optionalApi "io.cloudevents:cloudevents-spring:$cloudEventsVersion"
+        optionalApi "io.cloudevents:cloudevents-json-jackson:$cloudEventsVersion"
+
+        optionalApi("io.cloudevents:cloudevents-avro-compact:$cloudEventsVersion") {
+            exclude group: 'org.apache.avro', module: 'avro'
+        }
+        optionalApi "org.apache.avro:avro:$avroVersion"
+        optionalApi "io.cloudevents:cloudevents-xml:$cloudEventsVersion"
+        testImplementation 'tools.jackson.core:jackson-databind'
```

Comment:
Why do we need this?
I don't believe that `cloudevents-json-jackson` can work with Jackson 3.
So, just rely on whatever is their transitive dependency.

Reply:
Removed.   Artifact from a previous test.